Apps/README.md

```diff
@@ -54,43 +67,60 @@ This will show the structure we've created which looks like this:
 ```
 
 ## Deploy the entire app
-
 Now we can deploy the entire application with one command:
 
 ![user input](../images/userinput.png)
-> `fn deploy --all --local`
+>```sh
+> fn deploy --all --local
+>```
 
-Once the command is done we can examine the structure of the `myapp2`
-application.  First we can get a list of deployed applications. The `l` or `list` command, followed by `a`, `app` or `apps` displays the list of
-applications:
+Once the command is done we can examine the structure of the `myapp2` application.  First, get a
+list of deployed applications. The `l` or `list` command, followed by `a`, `app` or `apps` displays
+the list of applications:
 
 ![user input](../images/userinput.png)
-> `fn list apps`
+>```sh
+> fn list apps
+>```
+
+```txt
+NAME		ID				
+myapp2		01CT77FVBTNG8G00GZJ0000001	
+```
 
 You should see `myapp2` in the list of deployed applications.  We can
-then list application's by using the `l` or `list` command, followed by `r`, `route` or `routes`:
+then list application's triggers by using the `list` command, followed by `triggers`:
 
 ![user input](../images/userinput.png)
-> `fn list routes myapp2`
+>```sh
+> `fn list triggers myapp2`
+>```
+
+```cs
+FUNCTION    NAME            ID                          TYPE    SOURCE          ENDPOINT
+hello		hello-trigger	01CT77HZ2NNG8G00GZJ0000005	http	/hello-trigger  http://localhost:8080/t/myapp2/hello-trigger
+myapp2      myapp2-trigger  01CT77FWDFNG8G00GZJ000000   http    /myapp2-trigger http://localhost:8080/t/myapp2/myapp2-trigger
+```
 
 If you have previously set the `FN_REGISTRY` registry environment variable
 your Docker image names will be prefixed by it. Otherwise your output will look
 like:
 
 ```
-/       myapp2-root:0.0.2 localhost:8080/r/myapp2
-/hello  hello:0.0.2       localhost:8080/r/myapp2/hello
+fndemouser/hello        0.0.2       64049f5cfc52        10 minutes ago      15.6MB
```

Comment:
This table doesn't appear to format quite correctly - the top line is out of kilter.

Reply:
Otherwise looks pretty reasonable.

Reply:
I added a fix for the formatting.